### PR TITLE
bug fix remove whitespace on sqltext before execute

### PR DESCRIPTION
**Pull request by sfc-gh-ext-simba-dl**
bug fix for issue 130
https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/130

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -499,7 +499,7 @@ exports.construct_hostname = function (region, account)
  */
 exports.isPutCommand = function (sqlText)
 {
-  return (sqlText.substring(0, 3).toUpperCase() === "PUT");
+  return (sqlText.replace(/\s+/g, '').substring(0, 3).toUpperCase() === "PUT");
```
Comment (sfc-gh-igarish) [inline lib/util.js:502]: Do we need to replace white space from whole SQL or just to trim before and after the command? Better to use trim() kind of function.

**Comment by sfc-gh-igarish** [comment]
@sfc-gh-ext-simba-dl can you make sure, it works after the change?

**Comment by sfc-gh-dszmolka** [comment]
is there anything else needed here to merge this PR ?

**Review by sfc-gh-igarish — APPROVED** [review]
LGTM

**Comment by sfc-gh-igarish** [comment]
@sfc-gh-ext-simba-dl you need to bing this branch up to date with base branch.

**Pull request merged**